diploma-project/components/InputFieldTags.jsx

```diff
@@ -0,0 +1,62 @@
+import {useState} from "react";
+
+const InputFieldTags = ({tags, setTags}) => {
+    const [newTag, setNewTag] = useState('');
+
+    const handleTagChange = (e) => {
+        setNewTag(e.target.value);
+    };
+
+    const handleAddTag = () => {
+        if (newTag.trim() === '') {
+            return;
+        }
+
+        setTags([...tags, newTag]);
+
+        setNewTag('');
+    };
+
+    const handleRemoveTag = (tagToRemove) => {
+        // Filter out the tag to be removed from the current tags
+        const updatedTags = tags.filter((tag) => tag !== tagToRemove);
+        setTags(updatedTags);
+    };
+
+    return (
+        <div className="form-control space-y-5">
+            <label className="label">
+                <span className="label-text">Tags - <em>Not required</em></span>
+            </label>
+            <div>
+                <input
+                    type="text"
+                    value={newTag}
+                    onChange={handleTagChange}
+                    className="input input-bordered w-full pr-24"
+                />
+                <button type="button" className="btn btn-ghost absolute right-5" onClick={handleAddTag}>
```

Comment:
The handleAddTag function is only called when the button is clicked. You might want to allow the user to add a tag by pressing the Enter key as well. You can do this by adding an `onKeyPress` event handler to the input field.

Reply:
<kbd>onKeyPress</kbd> is deprecated, but I will see what I can do about this suggestion.

Reply:
Ah, yeah, I keep remembering old stuff. You can leave this for later, but it's something that is easily noticeable